modules/stanford_profile_styles/lib/scss/component/node/stanford_opportunity.scss

```diff
@@ -4,74 +4,292 @@
 .opportunity {
   @include responsive-spacing('margin-bottom', 6);
 
+  // adjusting layout for meantime
+  .su-opp-node-container-wrapper {
+    &.jumpstart-ui--two-column {
+      .left-region {
+        max-width: 904px;
+        margin: 0 auto;
+      }
+
+      .main-region {
+        max-width: 300px;
+
+        h2,
+        .su-opp-type,
+        .su-opp-course-code,
+        .su-opp-units,
+        time,
+        .su-opp-contact-name,
+        .su-opp-contact-email,
+        .su-opp-contact-phone,
+        .su-opp-contact-url,
+        .su-opp-learn-more,
+        .su-opp-cta-url {
+          @include responsive-spacing('margin-left', 1);
```

Comment:
Let's reduce the nesting as much as we can in all the css. At this point we are already 5 levels of specificity. This makes it very difficult to change in the future.